storage/connectors/redis/src/main/java/feast/storage/connectors/redis/retriever/RedisOnlineRetriever.java

```diff
@@ -45,6 +49,13 @@ public RedisOnlineRetriever(StatefulRedisConnection<byte[], byte[]> connection)
     this.syncCommands = connection.sync();
   }
 
+  public RedisOnlineRetriever(RedisConfig config) {
```

Comment:
Should this follow the BQ pattern of `RedisOnlineRetriever.fromConfig(config)`?

Reply:
Yip, I think you might have been reviewing the commit prior to this one. The change has been made.